app/api/domains/osu.py

```diff
@@ -1784,13 +1750,10 @@ async def get_osz(
     if no_video:
         map_set_id = map_set_id[:-1]
 
-    if USING_CHIMU:
-        query_str = f"download/{map_set_id}?n={int(not no_video)}"
-    else:
-        query_str = f"d/{map_set_id}"
+    query_str = f"{map_set_id}?n={int(not no_video)}"
 
     return RedirectResponse(
-        url=f"{app.settings.MIRROR_URL}/{query_str}",
+        url=f"{app.settings.MIRROR_DOWNLOAD_ENDPOINT.strip('/')}/{query_str}",
```

Comment:
We shouldn't allow for 2 different formats

Reply:
Afaik it is programming standard to always do this for compatibility, just like you use path combining tools instead of just concatenating strings

Reply:
the entire project disallows this. it should not be handled here. if you wish to do this, open up a separate PR and strip leading slashes on config READ so you only do it once

Reply:
I already suggested that but cmyui didn't want to do that tho it would help a lot since there has been like 20 people who always asked for help because they had an issue with the mirrors not working until we eventually found out they just had a / at the end

Reply:
either way this should be removed since it has nothing to do with this pr (also if you do end up opening a pr doing this, use rstrip instead)
```suggestion
        url=f"{app.settings.MIRROR_DOWNLOAD_ENDPOINT/{query_str}",
```